core/services/chainlink/testdata/config-invalid.toml

```diff
@@ -146,4 +146,8 @@ Enabled = true
 
 [P2P]
 [P2P.V2]
-Enabled = false
\ No newline at end of file
+Enabled = false
+
+[[EVM]]
+ChainID = '10200'
+ChainType = 'xdai' # This value is deprecated but should still not result in a validation error
```

Comment:
This file is meant to demonstrate _invalid_ configuration. Why are we updating it with a ChainType that is valid?

Reply:
Let's add a chain to one of the multi-chain files instead.

Reply:
Added a test for `xdai` to `wanings.xtar` instead: https://github.com/smartcontractkit/chainlink/blob/66e378925ca8c7a981cbc9b2a59c08d88d7e7783/testdata/scripts/node/validate/warnings.txtar#L393-L393